### PR TITLE
feat: add settings for primary vs secondary badge text color

### PR DESCRIPTION
**Pull request by tmchow**
Also adjust defaults for styles to be more complimentary colors

Default style:
image

Playful style:
image

### CODE REVIEW

File: source/settings.json [hunk 1]
```diff
@@ -273,10 +297,25 @@
...
       "default": "#C43F06"
     },
+    {
+      "variable": "badge_text_color_secondary",
+      "label": "Badge text color (secondary)",
+      "default": "#222222"
+    },
+    {
+      "variable": "badge_background_color_secondary",
+      "label": "Badge background color (secondary)",
```
Comment (nickendle) [inline source/settings.json:316]: Can probably ditch the 'color' on these labels

Comment (tmchow) [reply source/settings.json:316]: @nickendle are you sure? Other settings have the word color e.g. `Button background color` vs `Button background`

Comment (nickendle) [reply source/settings.json:316]: @tmchow  yeah i don't think it really makes sense to have the word color repeated across every color setting. I didn't realize older settings still had it (we have started to move away from including it altogether with the introduction of the new color settings panel, it's likely this just hadn't been updated yet) -- I'm in favor of removing all instances of it unless you have strong feelings for keeping it. Only having it on some of them feels weird though

Comment (tmchow) [reply source/settings.json]: @nickendle removed all instances of "color" from label

**Pull request merged**